api/src/aliases/StationId.ts

```diff
@@ -0,0 +1 @@
+export type StationId = number;
```

Comment:
What's this ?

Reply:
Alias. Create a type named `StationId` and let it be an int for now.
So we can use this type across the project and in case if we want to change that, this would be the only one instead of all the files that working with stationId

Reply:
Clear!